int8 id
float32 pid_kp
float32 pid_ki
float32 pid_kd
float32 pid_kdd
float32 pid_anti_windup
uint8 pid_en_pv_derivation
float32 mrft_beta
float32 mrft_relay_amp
float32 mrft_bias
float32 sm_alpha1
float32 sm_alpha2
float32 sm_h1
float32 sm_h2

